src/main/java/seedu/lifeasier/ui/Ui.java

```diff
@@ -166,10 +175,28 @@ public void showHelp() {
     }
 
     /**
-     * Returns read user command.
+     * Prompts continually for user command and checks if save delimiter has been added to prevent errors.
+     *
+     * @return User command without save delimiters.
      */
     public String readCommand() {
-        return conversation.nextLine();
+        boolean isCommandWithoutDelimiter = false;
+        String userInput = "";
+        do {
+            userInput = conversation.nextLine();
+
+            try {
+                if (userInput.contains("=-=")) {
+                    throw new SaveDelimiterException();
+                }
+                isCommandWithoutDelimiter = true;
+            } catch (SaveDelimiterException e) {
+                System.out.println(colourTextRed("Inputs cannot contain the character sequence: =-="));
+                System.out.println(colourTextCyan("Please re-enter your input:"));
+            }
+        } while (!isCommandWithoutDelimiter);
```

Comment:
Could this be simplified? It's a double negative.
```suggestion
        } while (isCommandWithDelimiter);
```

Reply:
Will change